src/data/finite/card.lean

```diff
@@ -92,6 +92,44 @@ by { haveI := fintype.of_finite α, haveI := fintype.of_surjective f hf,
 lemma card_eq_zero_iff [finite α] : nat.card α = 0 ↔ is_empty α :=
 by { haveI := fintype.of_finite α, simp [fintype.card_eq_zero_iff] }
 
+/-- NB: `nat.card` is defined to be `0` for infinite types. -/
+lemma card_le_of_injective' (f : α → β) (hf : function.injective f)
```

Comment:
```suggestion
lemma card_le_of_injective' {f : α → β} (hf : function.injective f)
```
Unless you have a reason to keep it explicit. Also, can you expand the docstring a little, explaining what the lemma says? Same below.